apps/condo/domains/notification/schema/Device.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Generated by `createschema notification.Device 'deviceId:Text; pushToken?:Text; pushTransport?:Select:firebase,apple,huawei; owner?:Relationship:User:SET_NULL; meta?:Json'`
+ */
+
+const { Text, Relationship, Select } = require('@keystonejs/fields')
+
+const { Json } = require('@core/keystone/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+
+const access = require('@condo/domains/notification/access/Device')
+
+const { PUSH_TRANSPORT_TYPES } = require('../constants/constants')
+
+const Device = new GQLListSchema('Device', {
+    schemaDoc: 'Used to describe device in order to be able to send push notifications via corresponding transport, depending on pushTransport. Device could be mobile or web based. ',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        deviceId: {
+            schemaDoc: 'Mobile/web device ID, which is used to identify device. One user can have many devices, and one device can be used by many users, one at a time.',
+            type: Text,
+            isRequired: true,
+            knexOptions: { isNotNullable: true },
+            kmigratorOptions: { null: false },
+        },
+
+        pushToken: {
+            schemaDoc: 'Used by transport services (FireBase, Apple, Huawei, etc.) to transfer push notifications to devices.',
```

Comment:
Please, tell more about lifecycle of the push token. Will it be assigned to device once be immutable or it can be changed later?

Reply:
fixed